.github/workflows/cmo-make-targets.yaml

```diff
@@ -81,7 +81,7 @@ jobs:
         author: ${{ env.USER }}
         committer: ${{ env.USER }}
         signoff: true
-        branch: automated-updates-master-${{ steps.branch.outputs.sandbox }}
+        branch: automated-updates-main-${{ steps.branch.outputs.sandbox }}
```

Comment:
do we need to change this as its just branch name in sandbox?
IIUC this is existing branch in rhobs and syncbot run updates it

Reply:
I think we do.
We're updating `main` now, and the bot can create/delete branches so it can start using that one.